.github/workflows/build-plugin-jira.yml

```diff
@@ -0,0 +1,33 @@
+name: Stuff to do CI for mattermost plugins
```

Comment:
@shigalovalexs I'm thinking this is out of scope of this PR. We're currently using CircleCI for the plugin projects. Also, if we were to add GitHub actions, we would want to support it holistically across projects rather than adding it to individual repos like this.